dist/api_data.py

```diff
@@ -1382,6 +1382,10 @@ def __ge__(self, other):
         warn, but not fail, in the case of errors that won't affect future
         behavior (for example, a leaked block)''',
         type='boolean'),
+    Config('hs_verify', 'false', r'''
+        Verify the history store''',
+        type='boolean')
+
```

Comment:
Thanks for making the comment on the ticket as well, I feel like however this discussion goes, (whether we go your way or the current way) it can be done after this PR as it's already quite hectic!
Will make sure Sulabh follows up on this :)

Reply:
I'm hesitant, as quite a few of your changes are dependent on answering that question. Is this ticket blocking forward progress elsewhere?